geode-wan/src/distributedTest/java/org/apache/geode/internal/cache/wan/wancommand/CreateDestroyGatewaySenderCommandDUnitTest.java

```diff
@@ -292,6 +295,7 @@ public void testCreateDestroyGatewaySenderWithGatewayTransportFilters() {
         + CliStrings.CREATE_GATEWAYSENDER__ALERTTHRESHOLD + "=100" + " --"
         + CliStrings.CREATE_GATEWAYSENDER__DISPATCHERTHREADS + "=2" + " --"
         + CliStrings.CREATE_GATEWAYSENDER__ORDERPOLICY + "=THREAD" + " --"
+        + CliStrings.CREATE_GATEWAYSENDER__GROUPTRANSACTIONEVENTS + "=false" + " --"
```

Comment:
Specifying the default value here is redundant.

Reply:
ok